templates/bridge-whatsapp/configmap.yaml

```diff
@@ -151,7 +151,14 @@ data:
         # the command prefix completely like in user management rooms is not possible.
         management: {{ .Values.bridges.whatsapp.relaybot.management }}
         # List of users to invite to all created rooms that include the relaybot.
-        invites: {{ .Values.bridges.whatsapp.relaybot.invites }}
+        {{- if .Values.bridges.whatsapp.relaybot.invites }}
+        invites:{{- if .Values.bridges.whatsapp.relaybot.invites }}
+        {{- range initial .Values.bridges.whatsapp.relaybot.invites }}
+        - {{ . | quote }}
+        {{- end }}
+        - {{ last .Values.bridges.whatsapp.relaybot.invites | quote }}
+        {{- end }}
+        {{- end }}
```

Comment:
I'm not quite following the templating here - you have two nested if statements that are equivalent? And then you're only ranging over `initial`, when `last` has the same format? Why not just

```
{{- if .Values.bridges.whatsapp.relaybot.invites }}
invites:
{{- range .Values.bridges.whatsapp.relaybot.invites }}
- {{ . | quote }}
{{- end }}
{{- end }}
```

Reply:
Sure, you are right...